src/app/app.server.ts

```diff
@@ -0,0 +1,16 @@
+import express, { Express, Router } from "express";
+import { router as breadRouter } from './bread/bread.router';
```

Comment:
Bread entity is not part of the challenge, this file should not be included. You can add the routers for other entities like the `Cake`.